src/scripts.js

```diff
@@ -330,7 +323,100 @@ fetchData('users/userData')
     });
   })
   .catch(error => console.log('hydrationData error'));
-
 })
+
 .catch(error => console.log('userData error'));
 // *****END OF .THEN FOR USERDATA**************************
+
+// Event listeners for show post-data form buttons
+$('.show__activity--btn').on('click', function() {
```

Comment:
When we get this figured out, we should try to aim to change these to named functions that we would call here